singer-io/tap-ringcentral
# Feature/detailed call log

# Description of change
The purpose of this PR is to enhance the existing dataset by adding endpoints for Call records. Additionally, by changing the nature of the GET request from "simple" to "Detailed" it opens the door for other in the community to pull PR's for additional fields on the Tap

https://github.com/singer-io/tap-ringcentral/issues/10

# Manual QA steps
 - Received valid 2XX positive code from Server
![image](https://user-images.githubusercontent.com/85885183/122991874-86ba7b00-d373-11eb-922a-bec3e23f9418.png)

- Got expected output of normal call and normal call + recording

![image](https://user-images.githubusercontent.com/85885183/122992239-e57ff480-d373-11eb-80d5-383aed317bc1.png)

# Risks
 - 
 
# Rollback steps
 - revert this branch
